docs/DeveloperGuide.md

```diff
@@ -303,17 +572,26 @@ Use case ends.
 
 3a. The given index is invalid
 
-> 3a1. AddressBook shows an error message <br>
+> 3a1. TaskManager shows an error message <br>
   Use case resumes at step 2
 
 {More to be added}
```

Comment:
Formatting is required for the use case. It seem quite inconsistent from the view section.